### PR TITLE
update to appengine-plugins-core 0.7.4

### PR DESCRIPTION
**Pull request by elharo**
@briandealwis

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.facets/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -8,8 +8,8 @@ Bundle-Vendor: %providerName
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
 Bundle-Localization: plugin
-Require-Bundle: com.google.cloud.tools.appengine;bundle-version="[0.6.8,0.7.0)",
- com.google.guava;bundle-version="[26.0.0,27.0.0)",
+Require-Bundle: com.google.cloud.tools.appengine;bundle-version="0.7.4",
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.facets/META-INF/MANIFEST.MF]: `[0.7.4,0.8.0)`?

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -18,21 +18,22 @@ Export-Package: com.google.cloud.tools.eclipse.appengine.libraries,
  com.google.cloud.tools.eclipse.appengine.libraries.model,
  com.google.cloud.tools.eclipse.appengine.libraries.persistence,
  com.google.cloud.tools.eclipse.appengine.libraries.repository
-Import-Package: com.google.cloud.tools.appengine.cloudsdk;version="[0.6.8,0.7.0)",
+Import-Package: com.google.cloud.tools.appengine.operations;version="0.7.4",
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/META-INF/MANIFEST.MF]: `[0.7.4,0.8.0)`?

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -18,21 +18,22 @@ Export-Package: com.google.cloud.tools.eclipse.appengine.libraries,
...
  com.google.cloud.tools.eclipse.util.service,
  com.google.cloud.tools.eclipse.util.status,
- com.google.cloud.tools.libraries;version="[0.6.8,0.7.0)",
- com.google.cloud.tools.libraries.json;version="[0.6.8,0.7.0)",
- com.google.common.annotations;version="[26.0.0,27.0.0)",
- com.google.common.base;version="[26.0.0,27.0.0)",
- com.google.common.cache;version="[26.0.0,27.0.0)",
- com.google.common.collect;version="[26.0.0,27.0.0)",
- com.google.common.util.concurrent;version="[26.0.0,27.0.0)",
+ com.google.cloud.tools.libraries;version="0.7.4",
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/META-INF/MANIFEST.MF]: `[0.7.4,0.8.0)`?

File: plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java [hunk 4]
```diff
@@ -277,9 +277,9 @@ public final void testBundleExecutionEnvironment() throws IOException {
   @Test
   public final void testGuavaImportVersions() throws IOException {
     checkDependencyDirectives(
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java]: We should have a similar test for `com.google.cloud.tools.appengine`

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/META-INF/MANIFEST.MF (hunk 2 above)
Comment (elharo) [reply plugins/com.google.cloud.tools.eclipse.appengine.libraries/META-INF/MANIFEST.MF]: 0.7.5 is likely to break the API again

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/META-INF/MANIFEST.MF (hunk 3 above)
Comment (elharo) [reply plugins/com.google.cloud.tools.eclipse.appengine.libraries/META-INF/MANIFEST.MF]: 0.7.5 is likely to break the API again

File: plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java (hunk 4 above)
Comment (elharo) [reply plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java]: I'm not sure this test has actually proved useful. I don't think it's caught anything yet.

**Comment by briandealwis** [comment]
> 0.7.5 is likely to break the API again

`;version="0.7.4",` is a lower-bound only, equivalent to `;version="[0.7.4,)"`.  We should be using `[0.7.4,0.7.4]`.

**Pull request merged**